dashboard/src/features/shared/routes.js

```diff
@@ -2,7 +2,7 @@ import { RoutingContainer } from 'features/shared/components'
 import { humanize } from 'utility/string'
 import actions from 'actions'
 
-const makeRoutes = (store, type, List, New, Show, options = {}) => {
+const makeRoutes = (store, type, List, New, Show, Update, options = {}) => {
```

Comment:
This shouldn't block the PR, but passing both an options object and having a bunch of optional components seems redundant. It might be better to take all arguments as part of `options`.